SwiftLeeds/Views/Local/LocalView.swift

```diff
@@ -57,6 +94,34 @@ struct LocalView: View {
             await model.loadData()
         }
     }
+
+    private func locationInfoView(category: Local.LocationCategory, location: Local.Location) -> some View {
+        VStack(spacing: 10) {
+            Image(uiImage: UIImage(systemName: category.symbolName) ?? UIImage(imageLiteralResourceName: category.symbolName))
+                .renderingMode(.template)
+                .frame(width: 44, height: 44)
```

Comment:
We'll need to have some aspect ratio stuff here - when you hard set the width and height like this, you'll be skewing(stretching? idk the word to use here) symbols that aren't squares

An easy way is below but theres many ways to achieve the same thing

```swift
.resizable()
.aspectRatio(contentMode: .fit)
.frame(maxWidth: 64, maxHeight: 64) // may have to pass alignment too depending on container
```

Reply:
We're using SFSymbols, and intent was for them all to be square

Reply:
All the ones we're using this year work well